pkgs/applications/misc/bambu-studio/default.nix

```diff
@@ -166,6 +166,10 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  postInstall = ''
+    rm $out/LICENSE.txt
```

Comment:
instead of removing it, how about moving it ?

also, there should be a comment explaining the reasoning

Reply:
You're right.
I've changed that!